src/components/Collection.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const Collection = ({active, collection, index, handleDelete, handleActive}) => {
```

Comment:
There is a lot of code duplication between these cards. We can definitely simplify these into a shared card.